src/index.html

```diff
@@ -5,9 +5,301 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="icon" href="images/favicon.png">
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="body">
+    <header class="header">
+      <div class="header__burger">
+        <input id="burger-toggle" type="checkbox">
+        <label class="header__burger-container" for="burger-toggle">
+          <span class="header__button-burger"></span>
+        </label>
+
+        <nav class="header__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item"><a href="#" class="nav__link">home</a></li>
```

Comment:
для удобства чтения лучше форматировать  так:

```suggestion
            <li class="nav__item">
              <a href="#" class="nav__link">
                home
              </a>
            </li>
```

Reply:
done